[2024-11-28T10:14:59.127+0700] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-28T10:14:59.132+0700] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: rocket_launch_pipeline.extract_images scheduled__2024-01-16T00:00:00+00:00 [queued]>
[2024-11-28T10:14:59.135+0700] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: rocket_launch_pipeline.extract_images scheduled__2024-01-16T00:00:00+00:00 [queued]>
[2024-11-28T10:14:59.135+0700] {taskinstance.py:2865} INFO - Starting attempt 1 of 4
[2024-11-28T10:14:59.143+0700] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_images> on 2024-01-16 00:00:00+00:00
[2024-11-28T10:14:59.144+0700] {standard_task_runner.py:72} INFO - Started process 80799 to run task
[2024-11-28T10:14:59.146+0700] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'rocket_launch_pipeline', 'extract_images', 'scheduled__2024-01-16T00:00:00+00:00', '--job-id', '136', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp0d7d2pk_']
[2024-11-28T10:14:59.147+0700] {standard_task_runner.py:105} INFO - Job 136: Subtask extract_images
[2024-11-28T10:14:59.167+0700] {task_command.py:467} INFO - Running <TaskInstance: rocket_launch_pipeline.extract_images scheduled__2024-01-16T00:00:00+00:00 [running]> on host Web-container
[2024-11-28T10:14:59.199+0700] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='rocket_launch_pipeline' AIRFLOW_CTX_TASK_ID='extract_images' AIRFLOW_CTX_EXECUTION_DATE='2024-01-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-16T00:00:00+00:00'
[2024-11-28T10:14:59.199+0700] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-28T10:14:59.199+0700] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-28T10:14:59.199+0700] {logging_mixin.py:190} INFO - Current task name:extract_images state:running start_date:2024-11-28 03:14:59.132637+00:00
[2024-11-28T10:14:59.200+0700] {logging_mixin.py:190} INFO - Dag name:rocket_launch_pipeline and current dag run status:running
[2024-11-28T10:14:59.200+0700] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-28T10:15:10.487+0700] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/fawzan/airflow/dags/dag.py", line 37, in extract_images
    download_image(url)
  File "/home/fawzan/airflow/dags/dag.py", line 40, in download_image
    response = requests.get(url)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/sessions.py", line 575, in request
    prep = self.prepare_request(req)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/sessions.py", line 484, in prepare_request
    p.prepare(
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/models.py", line 367, in prepare
    self.prepare_url(url, params)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/models.py", line 438, in prepare_url
    raise MissingSchema(
requests.exceptions.MissingSchema: Invalid URL 'None': No scheme supplied. Perhaps you meant https://None?
[2024-11-28T10:15:10.496+0700] {logging_mixin.py:190} INFO - Task instance in failure state
[2024-11-28T10:15:10.497+0700] {logging_mixin.py:190} INFO - Task start:2024-11-28 03:14:59.132637+00:00 end:2024-11-28 03:15:10.496228+00:00 duration:11.363591
[2024-11-28T10:15:10.497+0700] {logging_mixin.py:190} INFO - Task:<Task(PythonOperator): extract_images> dag:<DAG: rocket_launch_pipeline> dagrun:<DagRun rocket_launch_pipeline @ 2024-01-16 00:00:00+00:00: scheduled__2024-01-16T00:00:00+00:00, state:running, queued_at: 2024-11-28 03:14:54.277144+00:00. externally triggered: False>
[2024-11-28T10:15:10.497+0700] {logging_mixin.py:190} INFO - Failure caused by Invalid URL 'None': No scheme supplied. Perhaps you meant https://None?
[2024-11-28T10:15:10.497+0700] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=rocket_launch_pipeline, task_id=extract_images, run_id=scheduled__2024-01-16T00:00:00+00:00, execution_date=20240116T000000, start_date=20241128T031459, end_date=20241128T031510
[2024-11-28T10:15:10.509+0700] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-28T10:15:10.510+0700] {standard_task_runner.py:124} ERROR - Failed to execute job 136 for task extract_images (Invalid URL 'None': No scheme supplied. Perhaps you meant https://None?; 80799)
Traceback (most recent call last):
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/fawzan/airflow/dags/dag.py", line 37, in extract_images
    download_image(url)
  File "/home/fawzan/airflow/dags/dag.py", line 40, in download_image
    response = requests.get(url)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/sessions.py", line 575, in request
    prep = self.prepare_request(req)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/sessions.py", line 484, in prepare_request
    p.prepare(
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/models.py", line 367, in prepare
    self.prepare_url(url, params)
  File "/home/fawzan/airflow/.venv/lib/python3.10/site-packages/requests/models.py", line 438, in prepare_url
    raise MissingSchema(
requests.exceptions.MissingSchema: Invalid URL 'None': No scheme supplied. Perhaps you meant https://None?
[2024-11-28T10:15:10.550+0700] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-11-28T10:15:10.562+0700] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-28T10:15:10.563+0700] {local_task_job_runner.py:245} INFO - ::endgroup::
